locales/base.json

```diff
@@ -49,7 +49,8 @@
     "curve": "Exchange stablecoins with extremely efficient trading",
     "valora-rewards": "Earn rewards by learning about Celo",
     "toucan": "Tokenized carbon credits for climate action at scale",
-    "cred": "Get your free web3 Credit Score"
+    "cred": "Get your free web3 Credit Score",
+    "swftpro": "Simple, safe, & secure cross-chain bridge for BTC/ETH/BSC/CELO/AVAX/+"
```

Comment:
This doesn't align to our current dapp description guidelines which dictate starting with an action verb. Perhaps: Bridge assets onto BTC/ETH/BSC/CELO/AVAX/+ safely and securely"